README.md

```diff
@@ -44,6 +45,18 @@ development {
     defaultConsistencyLevel = 1
     replicationStrategy = "SimpleStrategy"
   }
+  consul {
+    host = "localhost"
+    host = ${?CONSUL_HOST}
```

Comment:
To keep examples short, probably omit the ENV VAR stuff.. the developer should be able to figure that stuff out.